### PR TITLE
Feature/191 implement qa form

### PR DESCRIPTION
**Pull request by Kosichenko**

### CODE REVIEW

File: slidez-fe-core/package.json [hunk 1]
```diff
@@ -89,5 +89,8 @@
     "react-scripts": "4.0.3",
     "tsconfig-paths-webpack-plugin": "^3.5.1",
     "typescript": "^4.1.2"
+  },
+  "prettier": {
+    "singleQuote": true
```
Comment (Anatolii-Ivanov) [inline slidez-fe-core/package.json]: Can you remove it?

File: slidez-fe-core/src/containers/sideBar/SideBar.tsx [hunk 2]
```diff
@@ -45,6 +54,10 @@ const SideBar = () => {
                         <img src={layout_icon} alt='layout'></img>
                     </a>
                 </span>
+                <button type='button' onClick={() => handleQAShow()}>
+                    Open Q&amp;A page
+                </button>
+                <Qa show={showQAModal} handleClose={() => handleQAClose()}></Qa>
```
Comment (Anatolii-Ivanov) [inline slidez-fe-core/src/containers/sideBar/SideBar.tsx]: I don't think that we need this button here

File: slidez-fe-core/src/pages/participant-page/Header.tsx [hunk 3]
```diff
@@ -1,19 +1,33 @@
...
             <div className='logo'>
                 <a href=''>
                     <img src={SlidezLogo} alt='Slidez Logo'></img>
                 </a>
             </div>
+            <button type='button' onClick={() => handleQAShow()}>
+                Open Q&amp;A page
+            </button>
+            <Qa show={showQAModal} handleClose={handleQAClose}></Qa>
             <div className='event-name'>{eventName}</div>
```
Comment (Anatolii-Ivanov) [inline slidez-fe-core/src/pages/participant-page/Header.tsx]: And here

File: slidez-fe-core/src/pages/participant-page/ParticipantPage.tsx [hunk 4]
```diff
@@ -29,8 +30,21 @@ const ParticipantPage = () => {
...
+    const handleQAShow = () => {
+        setShowQAModal(true)
+    }
+
     return (
         <div className='participant-page'>
+            <button type='button' onClick={() => handleQAShow()}>
+                Open Q&amp;A page
+            </button>
+            <Qa show={showQAModal} handleClose={() => handleQAClose()}></Qa>
```
Comment (Anatolii-Ivanov) [inline slidez-fe-core/src/pages/participant-page/ParticipantPage.tsx:47]: Alexandra added places to this button on poll component and empty page, please, add button there

File: slidez-fe-core/src/pages/qa/Qa.tsx [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+const Qa = (qaProps: QaProps) => {
+    const { handleClose, show } = qaProps
+    const classes = useStyles()
+    const [listQA, setListQA] = useState(MOCK_DATA)
+    const [isRecentSelected, setIsRecentSelected] = useState(true)
+
+    const handleSubmit = (textValue: string) => {
+        var now: string = new Date().toDateString()
+        const newQA = {
+            UUID: now,
```
Comment (Anatolii-Ivanov) [inline slidez-fe-core/src/pages/qa/Qa.tsx:47]: UUID should not be set on frontend, don't define it (nullable)

**Review by Anatolii-Ivanov — CHANGES_REQUESTED** [review]

File: slidez-fe-core/package.json (hunk 1 above)
Comment (Kosichenko) [reply slidez-fe-core/package.json]: Fixed

File: slidez-fe-core/src/containers/sideBar/SideBar.tsx (hunk 2 above)
Comment (Kosichenko) [reply slidez-fe-core/src/containers/sideBar/SideBar.tsx]: Fixed

File: slidez-fe-core/src/pages/participant-page/ParticipantPage.tsx (hunk 4 above)
Comment (Kosichenko) [reply slidez-fe-core/src/pages/participant-page/ParticipantPage.tsx:47]: It's cool! I'll will done it.

File: slidez-fe-core/src/pages/qa/Qa.tsx (hunk 5 above)
Comment (Kosichenko) [reply slidez-fe-core/src/pages/qa/Qa.tsx:47]: fixed

Comment (Kosichenko) [reply slidez-fe-core/src/pages/qa/Qa.tsx:47]: But than like button will work incorrect cause like depends on UUID.

**Review by Anatolii-Ivanov — APPROVED** [review]

**Review by alex-tiutiunnyk — APPROVED** [review]

**Pull request merged**